docs/quickstart.asciidoc

```diff
@@ -269,12 +288,21 @@ Login as the `elastic` user. The password can be obtained with the following com
 kubectl get secret quickstart-es-elastic-user -o=jsonpath='{.data.elastic}' | base64 --decode; echo
 ----
 
-[id="{p}-upgrade-deployment"]
-== Upgrade your deployment
+For a full description of each `CustomResourceDefinition` (CRD), refer to the <<{p}-api-reference>> or view the CRD files in the link:{eck_github}/tree/{eck_release_branch}/config/crds[project repository]. You can also retrieve information about a CRD from the instance. For example, describe the {kib} CRD specification via link:https://kubernetes.io/docs/reference/kubectl/generated/kubectl_describe/[`describe`]:
+
+[source,sh]
+----
+kubectl describe crd kibana
+----
+
+This completes the quickstart of deploying an {kib} instance on top of <<{p}-deploy-eck,the ECK operator>> and <<{p}-deploy-elasticsearch,deployed {es} cluster>>. We recommend continuing to <<{p}-upgrade-deployment,upgrading your deployment>>. For more {kib} configuration options, see <<kibana,Running {kib} on ECK>>.
```

Comment:
I don't think `{p}-upgrade-deployment` is valid because it is deleted from line 272.